Source/player.cpp

```diff
@@ -600,7 +600,7 @@ bool PlrHitMonst(Player &player, Monster &monster, bool adjacentDamage = false)
 	}
 
 	if (gbIsHellfire && HasAllOf(player._pIFlags, ItemSpecialEffect::FireDamage | ItemSpecialEffect::LightningDamage)) {
-		int midam = player._pIFMinDam + GenerateRnd(player._pIFMaxDam - player._pIFMinDam);
+		int midam = player._pIFMinDam + GenerateRnd(player._pIFMaxDam - player._pIFMinDam + 1);

```

Comment:
Isn't there a `randomIntBetween` function we can use here that already does this?

Reply:
Yeah, use `RandomIntBetween(player._pIFMinDam, player._pIFMaxDam)`. Closed ranges (where you add one to the roll to get a result back that potentially includes the max damage) are the default

Reply:
Fixed.  Thanks guys